plugins/flyte.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: flyte
+spec:
+  version: "v0.5.12"
+  homepage: https://github.com/lyft/flyte
+  shortDescription: kubectl-flyte is an command line tool for flyte that can be used as an extension to kubectl.
```

Comment:
"Kubectl-flyte is a command line tool" is redundant. Keep it short and descriptive. 

Right now this description does not really convey any meaning. It is already clear that this is a plugin and it's about Flyte. ￼￼